data_structures/src/builders.rs

```diff
@@ -29,9 +32,18 @@ pub const GENESIS: u64 = 0x0123_4567_89AB_CDEF;
 // BUILDER PUBLIC FUNCTIONS
 ////////////////////////////////////////////////////////////////////////////////////////
 impl Message {
+    /// Function to build Block message
```

Comment:
```suggestion
    /// Block message factory
```

Reply:
We can use this notation for every builder.

Reply:
We could even rename every `Whatever::build_message` as `Whatever::factory` or `Whatever::message_factory`. I'm 60% for it, 40% against.